src/Diagnostics.ModelsAndUtils/Attributes/SupportTopic.cs

```diff
@@ -14,14 +14,24 @@ public class SupportTopic : Attribute, IEquatable<SupportTopic>
         /// See <see href="http://aka.ms/selfhelppreview"/>
         public string Id;
 
+        /// <summary>
+        /// Sap Support Topic Id
+        /// </summary>
+        public string SapSupportTopicId;
```

Comment:
Should we use a property setter here and make sure that the supplied value is a GUID for Sap id's?

Reply:
Nazeef you mean that we should validate that it is a GUID before parsing SapId? 
I think it's fine as long as the SapId string in detector metadata is matching the SapId in the request. But I am open to change this if you think it is needed.

Reply:
Naah, it's fine.. I was just trying to ensure we catch a user who accidentally makes an error in Copy+Paste and then wonders why the detector is not being triggered. We can at least ensure that that the value is a GUID and avoid some percentage of issues.